packages/utils/test/string.test.ts

```diff
@@ -45,3 +49,60 @@ describe('isMatchingPattern()', () => {
     expect(isMatchingPattern([] as any, 'foo')).toEqual(false);
   });
 });
+
+// NOTE: These tests are copied (and adapted for chai syntax) to `string.test.ts` in `@sentry/browser`. The
+// base64-conversion functions have a different implementation in browser and node, so they're copied there to prove
+// they work in a real live browser. If you make changes here, make sure to also port them over to that copy.
+describe('base64ToUnicode/unicodeToBase64', () => {
+  const unicodeString = 'Dogs are great!';
+  const base64String = 'RG9ncyBhcmUgZ3JlYXQh';
+
+  test('converts to valid base64', () => {
+    expect(BASE64_REGEX.test(unicodeToBase64(unicodeString))).toBe(true);
```

Comment:
Isn't this assertion redundant when we already test for equality in the next lines?

Reply:
The equality test passing implies this one will, it's true, but not vice-versa. My thinking here is that if the equality test fails but this test passes, it narrows down the problem.